core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3608,6 +3608,13 @@ public void mount(AlluxioURI alluxioPath, AlluxioURI ufsPath, MountContext conte
         }
         mMountTable.checkUnderWritableMountPoint(alluxioPath);
 
+        MountInfo mountInfo = mMountTable.getMountTable().get(alluxioPath.getPath());
+        Map<String, String> propertyMap = context.getOptions().getPropertiesMap();
+        if (mountInfo != null && propertyMap != null
+            && Boolean.parseBoolean(propertyMap.get("remount"))) {
```

Comment:
per our discussion, let's change this to a new option in mount command

Reply:
done